build.go

```diff
@@ -427,7 +429,13 @@ func install(target target, tags []string) {
 
 	os.Setenv("GOOS", goos)
 	os.Setenv("GOARCH", goarch)
+
+	// The syso file doesn't affect non-Windows builds so it's fine to build it even when goos!=windows.
+	sysoPath := mustBuildSyso(cwd)
```

Comment:
Even if not strictly necessary, I would still wrap it in `if goos == "windows"` - it's clear and more concise than the comment. Using defer you can put `shouldCleanupSyso` into the same if block.

Reply:
Good idea! Hopefully I can blame completely forgetting `defer` exists on being sick :|